IsTheMartin/mrtnmrls
# Configure RSS description properly

## Summary

Configured `HtmlSanitizer` and `MarkdownIt`plugin to improve html content in description tags for RSS content.
Functions were created to improve code readability.

## Evidence

### Before
```
<description><![CDATA[<p>En casa tenemos plantada una <a href="https://es.wikipedia.org/wiki/Ficus_carica">higuera</a> (los higos son sus frutos), tiene aproximadamente unos cuatro años de edad. Recuerdo que plantamos una simple rama en la tierra, luego de unas semanas le salieron unas raíces y con el constante cuidado (riego y abono mayormente) se convirtió en un árbol. Llegó a crecer más de tres metros de altura y ha dado más de 100 higos en todo este tiempo. &lt;br /&gt; No obstante, en este invierno ha sido un poco duro para la higuera, dejó de dar frutos y las hojas se secaron y cayeron. Hace un par de semanas decidimos cortar algunas de sus ramas. &lt;br /&gt; &lt;figure class="m-o" data-template="with"&gt; &lt;div class="m-m"&gt; &lt;img src="/images/posts/common-fig-tree.webp" alt="Higuera en invierno" class="img" loading="lazy" style="aspect-ratio: 512 / 512;" /&gt; &lt;/div&gt; &lt;figcaption&gt;Higuera en invierno&lt;/figcaption&gt; &lt;/figure&gt; &lt;br /&gt; A día de hoy, el invierno está por terminar y cada vez hace más calor en el ambiente, le han crecido varias ramas y hojas muy sanas. &lt;br /&gt; &lt;figure class="m-o" data-template="with"&gt; &lt;div class="m-m"&gt; &lt;img src="/images/posts/sprouting-common-fig-tree.webp" alt="Hojas de higuera retoñando" class="img" loading="lazy" style="aspect-ratio: 512 / 512;" /&gt; &lt;/div&gt; &lt;figcaption&gt;Hojas de higuera retoñando&lt;/figcaption&gt; &lt;/figure&gt; &lt;br /&gt; Espero que continue creciendo y en otoño salgan algunos deliciosos higos.</p> ]]></description>
``` 
### After
```
<description><![CDATA[En casa tenemos plantada una <a href="https://es.wikipedia.org/wiki/Ficus_carica">higuera</a> (los higos son sus frutos), tiene aproximadamente unos cuatro años de edad. Recuerdo que plantamos una simple rama en la tierra, luego de unas semanas le salieron unas raíces y con el constante cuidado (riego y abono mayormente) se convirtió en un árbol. Llegó a crecer más de tres metros de altura y ha dado más de 100 higos en todo este tiempo. <br /> No obstante, en este invierno ha sido un poco duro para la higuera, dejó de dar frutos y las hojas se secaron y cayeron. Hace un par de semanas decidimos cortar algunas de sus ramas. <br /> <img src="https://mrtnmrls.com/images/posts/common-fig-tree.webp" alt="Higuera en invierno" loading="lazy" style="aspect-ratio:512 / 512" /> Higuera en invierno <br /> A día de hoy, el invierno está por terminar y cada vez hace más calor en el ambiente, le han crecido varias ramas y hojas muy sanas. <br /> <img src="https://mrtnmrls.com/images/posts/sprouting-common-fig-tree.webp" alt="Hojas de higuera retoñando" loading="lazy" style="aspect-ratio:512 / 512" /> Higuera retoñando <br /> Espero que continue creciendo y en otoño salgan algunos deliciosos higos. ]]></description>
``` 

## Concern Information

To test please follow the next steps:

1. Navigate to RSS page
2. Observe `<description>` content

## Issue ticket

- Resolves #37 

## Check list

- [ x ] I did add reviewers to this PR
